action.yml

```diff
@@ -28,6 +28,12 @@ inputs:
   command:
     description: 'The command used by ECS to start the container image'
     required: false
+  task-role-arn:
+    description: 'The ARN of the IAM role that the ECS container will assume'
+    required: false
+  execution-role-arn:
+    description: 'The ARN of the IAM role that the ECS task execution role will assume'
```

Comment:
This description can be made more explicit, as it could confuse some users since it's self-referencing.
Suggested reword: 'The ARN of the IAM role that the task definition will assume to run the task'